src/Sarif.Viewer.VisualStudio.Core/SarifCommandPackage.vsct

```diff
@@ -132,6 +132,14 @@
           <ButtonText>Clear all SARIF results</ButtonText>
         </Strings>
       </Button>
+      <Button guid="guidSarifErrorListCmdSet" id="cmdidSuppressResultCommand" priority="0x0100" type="Button">
+        <Parent guid="guidSarifErrorListCmdSet" id="ErrorListContextMenuGroup" />
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <CommandFlag>DynamicVisibility</CommandFlag>
+          <Strings>
+            <ButtonText>Suppress result in log file</ButtonText>
```

Comment:
>Suppress result in log file [](http://example.com/codeflow?start=24&length=27)

question, I see two kind of string
"Suppress result in log file"
$"Suppress issue {this.sarifErrorListItem.Rule.Id} in SARIF file"

should we also say in same format like
"Suppress issue in SARIF file"

Reply:
Other menu items for error list use same term 'result', to be consistent with context, changed it to "Suppress result in SARIF file"